source/client/process_impl.h

```diff
@@ -114,6 +114,15 @@ class ProcessImpl : public Process, public Envoy::Logger::Loggable<Envoy::Logger
   std::vector<StatisticPtr>
   mergeWorkerStatistics(const std::vector<ClientWorkerPtr>& workers) const;
   void setupForHRTimers();
+  /**
+   * If there are sinks configured in bootstrap, populate stats_sinks with sinks
+   * created through NighthawkStatsSinkFactory and add them to store_root_.
+   *
+   * @param bootstrap the bootstrap configuration which include the stats sink configuration.
+   * @param stats_sinks a Sink list to be populated.
+   */
+  void setupStatsSinks(envoy::config::bootstrap::v3::Bootstrap& bootstrap,
```

Comment:
Looks like `envoy::config::bootstrap::v3::Bootstrap&` may be a read-only argument, if it is so, can we make it a const reference?

```cc
void setupStatsSinks(const envoy::config::bootstrap::v3::Bootstrap& bootstrap, ... )
```

Reply:
Done.